pennylane/_qubit_device.py

```diff
@@ -797,6 +800,15 @@ def statistics(
 
         return results
 
+    def is_lightning_device(self):
```

Comment:
Could this be private?  Probably doesn't matter much at this stage.

Reply:
Yes, will do.